org.eclipse.scanning.api/src/org/eclipse/scanning/api/annotation/ui/FieldValue.java

```diff
@@ -249,29 +249,29 @@ private static Method getMethod(Class<?> clazz, String name) throws IllegalAcces
 		}
 		return null;
 	}
-	
 
-	
+
+
 	public boolean isModelField(String name) throws NoSuchFieldException, SecurityException {
         return isModelField(model, name);
 	}
 
 	public static boolean isModelField(Object model, String name) throws NoSuchFieldException, SecurityException {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.lang.SecurityException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)